modules/integration/tests-ui-integration/src/test/java/org/wso2/am/integration/ui/tests/APIWsdlDownloadTestCase.java

```diff
@@ -60,11 +54,20 @@
     private ServerConfigurationManager configManagerDB, configManagerMasterDatasource, configManagerReg;
     private WebDriver driver;
     private WebDriverWait wait;
+    private final String SUPER_TENANT_DOMAIN_NAME = "carbon.super";
+    private final String SUPER_TENANT_USERNAME = "admin";
+    private final String SUPER_TENANT_PASSWORD = "admin";
+    private final String TENANT_DOMAIN_NAME = "wso2.com";
```

Comment:
You may use multi tenant utils to get super tenant username domain etc. No need to hard code them here.